binancial/data/get_trades_historical.py

```
@@ -2,18 +1,17 @@
+def get_trades_historical(client, symbol="BTCUSDT"):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_trades_historical`

(ANN201)